semperfiwebdesign/all-in-one-seo-pack
# Updated sitemaps.org link to open in new tab with noopener and noreferrer

Issue #2399

## Proposed changes

Updated sitemaps.org link to open in new tab with noopener and noreferrer.

## Types of changes

- Improves existing code

## Checklist


## Testing instructions

Go to the XML Sitemap and check the sitemaps.org link to see if it works and opens in a new tab.
